jobs/distributable_lib.tcl

```diff
@@ -289,10 +295,11 @@ proc check_licenses {portName variantInfo} {
             }
 
             if {!$any_good} {
-                return [list 1 "\"$portName\" is not distributable because its dependency \"$aPort\" has license \"$lic\" which is not known to be distributable"]
+                return [list 1 "\"$portName\" is not distributable because its dependency \"$aPort\" has license \"$lic\" which is not known to be distributable: $parentPath"]
```

Comment:
Does `set parentPath [join $parentPath " -> "]` need to be here too?

Reply:
Fixed; I inlined the change.